zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -1869,6 +1869,78 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
+  /resources/{resourceKey}:
+    get:
+      tags:
+        - Resource
+      summary: Get resource
+      description: |
+        Returns a deployed resource.
+        :::note
+        Currently, this endpoint only supports RPA resources.
+        :::
+      parameters:
+        - name: resourceKey
+          in: path
+          required: true
+          description: The assigned key of the resource, which acts as a unique identifier for this resource.
```

🔧 All keys in Camunda 8 are assigned. Let's align this to the other usage of the term key in resources: 

```suggestion
          description: The unique key identifying the resource.
```

---

I took the description from the `processDefinitionKey` in [this endpoint](https://docs.camunda.io/docs/next/apis-tools/camunda-api-rest/specifications/get-process-definition/), but I'm happy to use a shorter one.